appliances/backup_and_restore.md

```diff
@@ -35,17 +35,22 @@ Before adding the cron job it would be best practice to mount an external storag
 # Restore
 Restoring a backup of the appliance will be done using SSH or the console.
 
-1. Download a backup TGZ and place it in `/`
-2. On the command line:
+1. Set the hostname on the new appliance.  On the command line:
+   ```bash
+     hostnamectl hostname your-new-hostname.example.com
+     echo "your.ip.address your-new-hostname.example.com your-new-hostname" > /etc/hosts
```

Comment:
Probably should be `>> /etc/hosts` so it doesn't wipe out the `127.0.0.1 localhost, etc...`

Reply:
We could probably add a command to get their IP automatically, `ip -4 -br addr show eth0 | awk '{print $3}'` just need a little regex to drop the `/subnet`